x-pack/test/cases_api_integration/security_and_spaces/tests/trial/delete_sub_privilege.ts

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CASES_URL } from '@kbn/cases-plugin/common';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+import { getPostCaseRequest, postCommentUserReq } from '../../../common/lib/mock';
+import {
+  deleteCasesByESQuery,
+  deleteCasesUserActions,
+  deleteComments,
+  createCase,
+  deleteCases,
+  createComment,
+  getCase,
+  superUserSpace1Auth,
+  findCases,
+  deleteComment,
+  getSpaceUrlPrefix,
+  updateComment,
+} from '../../../common/lib/utils';
+import { superUser, secOnlyDelete } from '../../../common/lib/authentication/users';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const es = getService('es');
+
+  describe('user with deletion sub privilege', () => {
```

Comment:
I was getting errors when the `basic` license attempted to use the secOnlyDelete user. I believe this was because the sub privileges are not available in basic license. So I move the tests to trail specifically.